packages/module/src/ShortcutGrid/ShortcutGrid.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { createUseStyles } from 'react-jss';
+import Shortcut, { ShortcutProps } from '../Shortcut/Shortcut';
+import { Grid, GridItem, GridItemProps, GridProps } from '@patternfly/react-core';
+
+export interface ShortcutGridProps extends GridProps {
+  /** Array of shortcuts to be displayed in the grid */
+  shortcuts: ShortcutProps[];
+  /** Shortcut GridItem props */
+  gridItemProps?: GridItemProps
+}
+
+const useStyles = createUseStyles({
+  shortcutItem: {
+    textAlign: 'right',
```

Comment:
This seems like something we could avoid using if we use a PF layout or modifier class.

Reply:
@nicolethoen is it possible to set the alignment somehow for the grid itself or do I need any other layout?
I see only modifier class for flex "pf-m-align-right" or the util class "pf-v5-u-text-align-right" which we probably don't want to use ^^

Reply:
I'm asking our CSS devs

Reply:
it seems the writing the css style out here is probably the best 👍🏻